target-platform/org.eclipse.kura.windows.service/src/main/c/main.c

```diff
@@ -34,6 +35,25 @@ char KuraJavaArgs[] = "java \
 -console \
 -consoleLog \
 ";
+#else
+char KuraJavaArgs[] = "java \
+-Xms256m -Xmx256m -Dkura.os.version=win32 \
+-Dkura.arch=x86 \
+-Dtarget.device=windows \
+-Declipse.ignoreApp=true \
+-Dkura.home=%KURA_HOME% \
+-Dkura.data.dir=%KURA_DATA% \
+-Dkura.configuration=file:%KURA_HOME_URI%/kura.properties \
+-Dkura.custom.configuration=file:%KURA_HOME_URI%/kura_custom.properties \
+-Ddpa.configuration=%KURA_HOME%\\dpa.properties \
+-Dlog4j.configuration=file:%KURA_HOME_URI%/log4j.properties \
+-Djdk.tls.trustNameService=true \
+-jar \"%~dp0\\plugins\\org.eclipse.osgi_3.8.1.v20120830-144521.jar\" \
```

Comment:
Is it a wise idea to hardcode that in a C file? Looking forward to the Equinox 3.11.0 upgrade, this will cause some major issues because the final target can no longer be built by just switching a maven profile.